staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -161,7 +160,7 @@ var (
 					Queuing: &flowcontrol.QueuingConfiguration{
 						Queues:           64,
 						HandSize:         6,
-						QueueLengthLimit: 1000,
+						QueueLengthLimit: 10,
```

Comment:
10 seems a bit on the small side, if we consider that many of our requests, especially read ones, are served in < 1ms. I think 50 or 100 is an OK starting point? (Apologies if we talked about this at the meeting and I said something different, I can't remember!)

Reply:
yeah we talked about that in the meeting and agreed the current queue lengths are exaggeratedly long but we didnt decide another actual number. applied 50 instead.